go/client/cmd_dump_keyfamily.go

```diff
@@ -48,21 +63,31 @@ func (v *CmdDumpKeyfamily) Run() (err error) {
 	if !currentStatus.LoggedIn {
 		return fmt.Errorf("Not logged in.")
 	}
-	myUID := currentStatus.User.Uid
+
+	var UID keybase1.UID
+	if v.user != "" {
+		res := v.G().Resolver.Resolve(v.user)
```

Comment:
Using `Resolver` means we're trusting the server to return the correct UID to us. Is that bad? Maybe the server could swap in a UID for a user with a suspiciously similar username that was under the server's control. I'm just being paranoid, but if it's easy to load user directly by username, it might be marginally better to do it that way?